crates/humanode-runtime/src/tests/fixed_supply.rs

```diff
@@ -160,106 +182,32 @@ fn total_issuance_transaction_payment_validate() {
 }
 
 #[test]
-fn total_issuance_evm_withdraw() {
```

Comment:
I think this one should still remain, but change a bit to show there is no withdrawal possible

Reply:
https://github.com/humanode-network/humanode/pull/673/commits/d6a0a80f1277a57af68eacbb7801673dd5dff9e6